setup/win32/Makefile

```diff
@@ -27,8 +25,11 @@ server_clean:
 	rm -rf $(SERVER_DIRECTORY)/.cyg*
 
 allinone: server_clean
-	#(cd $(SERVER_DIRECTORY)/setup/win32 && $(LAUNCH_PY2EXE_SERVICE))
-	#(cd $(SERVER_DIRECTORY) && $(LAUNCH_PY2EXE))
+	cp $(SERVER_DIRECTORY)/requirements.txt $(WINPY_DIR)/
+	-(cd $(WINPY_DIR) && ./python.exe -m pip install --upgrade pip)
+	-(cd $(WINPY_DIR) && cat requirements.txt | while read PAC ; do Scripts/pip3.exe install "$${PAC}" ; done)
```

Comment:
use `pip3.exe install -r $(SERVER_DIRECTORY)/requirements.txt`  [1]

And you don't need to copying it to $(WINPY_DIR)

[1] see https://pip.pypa.io/en/stable/user_guide/#requirements-files

Reply:
I'm afraid that SERVER_DIRECTORY will point to ../../ relatively to WINPY_DIR because I did a `cd`.
That's the reason why I copy the file.

Reply:
doesn't matter where the file is located.